web/js/containers/sidebar/layer.js

```diff
@@ -54,11 +56,14 @@ class Layer extends React.Component {
       isMobile
     } = this.props;
     if (!lodashIsEmpty(renderedPalette)) {
-      const isRunningData = !!runningObject;
+      const isRunningData = compare.active
+        ? (runningObject || {}).layerGroupName === layerGroupName
```

Comment:
My only thought is that don't you need to pass `layerGroupName` in the runningObject. `compare.active ? compare.activeString === layerGroupName` here should do the trick?

Reply:
Yep, good catch!